src/RacingController.js

```diff
@@ -0,0 +1,47 @@
+import InputView from './InputView';
+import Validation from './utils/Validation';
+import OutputView from './OutputView';
+import Cars from './Cars';
+
+export class Game {
+  async play() {
+    const carNameArray = await Game.getCarNamesArray();
+    const cars = new Cars(carNameArray);
+    const tryCount = await Game.getTryCount();
+    Game.moveCars(cars, tryCount);
+    OutputView.printWinner(cars.findWinner());
```

Comment:
선언부과 실행부 사이에 개행이 있으면 더 눈에 잘 들어올 것 같아요 :)

```suggestion
    const carNameArray = await Game.getCarNamesArray();
    const cars = new Cars(carNameArray);
    const tryCount = await Game.getTryCount();
    
    Game.moveCars(cars, tryCount);
    OutputView.printWinner(cars.findWinner());
```